src/core.js

```diff
@@ -103,10 +103,21 @@ export default class Core {
   }
 
   on(query, events, callback) {
-    const el = _element(query)
-    if (!el) return
+    let elements = _elements(query)
+    if (!elements) return
+
+    if (elements.length == undefined) elements = [elements]
```

Comment:
Maybe this will be cleaner using `Array.isArray(element)`, example: https://github.com/ralixjs/ralix/blob/979e5f354699e1c38bd013575e4ed0ac7c00ecbf/src/core.js#L190